apps/wallet/src/ui/app/pages/home/apps/index.tsx

```diff
@@ -61,7 +48,7 @@ function AppsPage() {
 				<section>
 					<FiltersPortal tags={allFilterTags} />
 					<Routes>
-						<Route path="/*" element={<AppsPlayGround />} />
+						<Route path="/:tagName" element={<AppsPlayGround />} />
 						<Route path="/connected" element={<ConnectedAppsCard />} />
```

Comment:
Not sure how pattern matching here works, does the order matter?

Reply:
yes, it does, fixed this